common/db.go

```diff
@@ -113,8 +121,14 @@ var schema = `
 	CREATE INDEX IF NOT EXISTS idx_blockheaders_number ON blockheaders(number);
 `
 
-func NewDBOrch(serviceURI string, orchAddr string) *DBOrch {
-	return &DBOrch{ServiceURI: serviceURI, EthereumAddr: orchAddr}
+func NewDBOrch(serviceURI string, orchAddr string, pricePerPixel int64, activationRound int64, deactivationRound int64) *DBOrch {
```

Comment:
Since `EthereumAddr` is the primary key in the table and it is always required when constructing a `DBOrch` to be passed into `UpdateOrch()`, it might be helpful to specify `orchAddr` as the first argument here and have the optional arguments follow it.

Reply:
If these args are optional then they should be put into a struct. That would also help make the sig less cumbersome.

Reply:
Ah right - the above comment made me realize that in the cases where we do not want to set certain values in `DBOrch` since they are optional we can just directly create the struct and only specify the fields we care about (i.e. `&DBOrch{orchAddr: orchAddr, activationRound: 100}`) instead of using this constructor. Any fields not specified will just take on their default value (in this case empty strings or 0 since each field is either a `string` or `int64`).

Reply:
I guess usage of the constructor vs literal is up to the caller of `db.SelectOrchs` and `db.OrchCount` , or do you want to get rid of the `NewDBOrch` function altogether?

Reply:
I'm fine with keeping the `NewDBOrch()` function. The comment about using the literal to avoid specifying optional args was just something that came to mind for the other PRs that build on top of this one, but I see you've already switched to using the literal in those PRs.